include/revng/Model/QualifiedType.h

```diff
@@ -34,6 +34,8 @@ TUPLE-TREE-YAML */
 class model::QualifiedType : public model::generated::QualifiedType {
 public:
   using generated::QualifiedType::QualifiedType;
+  QualifiedType(decltype(UnqualifiedType) Unqualified) :
+    model::generated::QualifiedType(std::move(Unqualified), {}) {}
```

Comment:
Do you really needs this? I'm significantly changing how we instantiate `QualifiedType` in the inline primitives branch, so I'm a bit wary of introducing new ways.

Reply:
I'll double check, but I should be able to feasible get rid of it. Not that it matters much now that `QualifiedType` is finally getting reworked.